pyformance/registry.py

```diff
@@ -126,7 +126,7 @@ def meter(self, key, tags=None):
     def create_sink(self):
         return None
 
-    def timer(self, key, tags=None):
+    def timer(self, key, tags=None, sample=None):
```

Comment:
```suggestion
    def timer(self, key, tags=None, sample_manager=None):
```

Reply:
I was going for consistency with the library as it is. The property of `Timer` and the argument to the `Timer` constructor (and `Histogram` etc.) are called `sample`. If I change it here, should I change it all the way down?

Reply:
sounds like it is the correct way to go but if that's the case than keeping it in the same convention is ok for us now I suppose. 
resolving.